subsys/usb/class/netusb/netusb.c

```diff
@@ -23,13 +23,11 @@
 #include "netusb.h"
```

Comment:
Could you explain also in commit message why we do this change in this "usb:netusb: Remove multi-function array" patch?